src/components/GradientView.tsx

```diff
@@ -0,0 +1,22 @@
+import { LinearGradient } from './Gradient';
+import React, { ReactChild } from 'react';
+import { ViewStyle } from 'react-native';
+
+import { GradientVariants } from 'consts';
+import { gradients } from 'styles';
+
+interface Props {
+  children?: ReactChild;
+  variant: GradientVariants;
```

Comment:
great to have enum! what do you think about exporting the enum from component? one import less for you. 
```
enum GradientVariant {
  Primary = 'Primary',
  Secondary = 'Secondary',
}
export class GradientView extends ... {
   static Variant = GradientVariant
...
}
```
and usage:
```
<GradientView variant={GradientView.Variant.Primary} />
```

by the way, it's good practice to have enum names as singular -> `GradientVariant` instead of variants

Reply:
Great tip. Thanks.